sdk/core/core-amqp/changelog.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-preview.2.0 - 16th July, 2019
+
+- `sendRequest()` function in the `RequestResponseLink` now excludes default retries and leaves it up to the users to implement it as necessary.
```

Comment:
There are other changes that went into preview 2 of core-amqp, we can cover the changelog at one go in the end. Either revert the changes to this file or replace the date with `Coming Soon...` so that it doesnt look like preview 2 has already been shipped